tests/integration/components/ancestry-display/component-test.ts

```diff
@@ -0,0 +1,112 @@
+import EmberObject from '@ember/object';
+import Service from '@ember/service';
+import { render } from '@ember/test-helpers';
+import I18N from 'ember-i18n/services/i18n';
+import { setupRenderingTest } from 'ember-qunit';
+import { TestContext } from 'ember-test-helpers';
+import faker from 'faker';
+import hbs from 'htmlbars-inline-precompile';
+import { module, test } from 'qunit';
+
+const i18nStub = Service.extend({
+    translations: EmberObject.create({
+        general: {
+            ellipsis: '...',
+        },
+    }),
+
+    t(key: string): string {
+        // @ts-ignore
+        return this.get('translations').get(key);
+    },
+});
+
+interface ThisTestContext extends TestContext {
+    i18n: I18N;
+}
+
+module('Integration | Component | ancestry-display', hooks => {
+    setupRenderingTest(hooks);
+
+    function createNode(extra?: object): any {
+        const node: any = EmberObject.create({
```

Comment:
You should be able to use the mirage factories here. [Here's an example](https://github.com/CenterForOpenScience/ember-osf-web/blob/a6d08032087dd33e16a5a09ae9c5e5b36c3b221b/tests/unit/models/osf-model-test.ts#L20)

```suggestion
        const node: Node = server.create('node');
```